plotly_express/_core.py

```diff
@@ -295,126 +306,139 @@ def configure_axes(args, constructor, fig, axes, orders):
         go.Choropleth: configure_geo,
     }
     if constructor in configurators:
-        fig.update(layout=configurators[constructor](args, fig, axes, orders))
+        configurators[constructor](args, fig, orders)
```

Comment:
configurators now update the layout in place rather than returning a new layout

Reply:
I don't mind (this is how it originally was!) but why this change now?

Reply:
This was because I wanted to be able to use `update_(x|y)axes` inside the `configure_cartesian_axes`.

Reply:
makes sense!